test/unit/api_spec.js

```diff
@@ -2926,6 +2926,29 @@ describe("api", function () {
       await loadingTask.destroy();
     });
 
+    it("write an highlight annotation and its popup deletion", async function () {
```

Comment:
Nit: Maybe reword this a tiny bit?
```suggestion
    it("write an highlight annotation and delete its popup", async function () {
```